src/content/wiki/items/tag_tool.mdoc

```diff
@@ -60,16 +60,21 @@ The following tags are currently implemented (more may be added from time to tim
+    | `sit`          | (leisure)                           | 0-many      | A "chair" where colonists will occasionally sit                                                                                           |
+    | `cobble`       | {% building name="miner" /%}        | Always 1    | The block behind the top-most ladder at the start of the mineshaft (traditionally cobblestone, but may be something else)                 |
+    | `ladder`       | {% building name="miner" /%}        | Always 1    | The top-most ladder at the start of the mineshaft                                                                                         |
+    | `bamboo`       | {% building name="plantation" /%}   | level-based | A block that bamboo can be planted on ([legacy](/wiki/tutorials/schematics#plantation-fields))                                            |
+    | `cactus`       | {% building name="plantation" /%}   | level-based | A block that cactus can be planted on ([legacy](/wiki/tutorials/schematics#plantation-fields))                                            |
+    | `sugar`        | {% building name="plantation" /%}   | level-based | A block that sugarcane can be planted on ([legacy](/wiki/tutorials/schematics#plantation-fields))                                         |
+    | `portal`       | {% building name="netherworker" /%} | Always 1    | An obsidian block in the bottom of the nether portal (with air above)                                                                     |
+    | `vault`        | {% building name="netherworker" /%} | 0-1         | A block in an enclosed room that the nether miner can stand on while "away"                                                               |
+    | `work`         | {% building name="archery" /%}      | level-based | The block the {% worker name="archer" /%} in Training stands on to shoot at targets (Note: If no tags are used, they look for glowstone.) |
+    | `job=miner`    | {% building name="miner" /%}        | Always 1    | Sets the mine's worker to be only {% worker name="miner" /%}                                                                              |
+    | `job=quarrier` | {% building name="miner" /%}        | Always 1    | Sets the mine's worker to be only {% worker name="quarrier" /%}                                                                           |
+    | `job=knight`   | {% building name="guardtower" /%}   | Always 1    | Sets the Guard Tower's worker to be only {% worker name="knight" /%}                                                                      |
+    | `job=ranger`   | {% building name="guardtower" /%}   | Always 1    | Sets the Guard Tower's worker to be only {% worker name="ranger" /%}                                                                      |
```

Comment:
Unfortunately we don't use the name "ranger" on the wiki, has to be "archer"

Reply:
I was told by @MotionlessTrain that the internal name for the archer for the purposes of the tag is ranger, not archer, hence why the tag is job=ranger

Reply:
`job=ranger` is fine, however the worker reference needs to be `{% worker name="archer" /%}`

Reply:
ohhh wait! I see what you mean

Reply:
I guess it's a good reminder because I wanted to change that for 1.21 because it's silly

Reply:
my biggest concern is that I had to add ---- to expand columns to fit tags

Reply:
The formatting in those tables does not strictly matter, if spacings are off Markdown will still know how to render the table, the only reason the spacings are neat like that is because when I edit locally I have plugins to neatly format tables.

Reply:
> I guess it's a good reminder because I wanted to change that for 1.21 because it's silly

ranger was the original name for the archer as far as I know. and tags now use it so I don't know if it is wise to change it

Reply:
For 1.21 it's too late now, and changing the tags is a one time small update, so that's no big deal.

The issue is in the naming of all the jobs in that regard.
- Knight = `knight`
- Archer = `ranger`
- Knight in Training = `combattraining`
- Archer in Training = `archertraining`

Literally makes no sense.

Reply:
ahh ok. I had a feeling that the spacings were that way to make it look neat on github but wasn't 100% sure.